### DISCUSSION TITLE
StreamListener on_limit

### DISCUSSION THREAD

**Discussion by franklincm** [tweepy/tweepy#1285] [Questions] [answered]
I apologize if this is obvious, but what is a limit notice as handled by `StreamListener.on_limit`? 

I get them with various `track` values, but can't seem to find anything on the twitter docs yet about what the values mean. Thanks for your time.

**Comment by Harmon758** [comment, marked as answer]
> Limit notices (limit)
> These messages indicate that a filtered stream has matched more Tweets than its current rate limit allows to be delivered. Limit notices contain a total count of the number of undelivered Tweets since the connection was opened, making them useful for tracking counts of track terms, for example. Note that the counts do not specify which filter predicates undelivered messages matched.

https://developer.twitter.com/en/docs/tweets/filter-realtime/guides/streaming-message-types
